content/authorization.md

```diff
@@ -382,6 +383,20 @@ where the audience is provided:
 &aud=https%3A%2F%2Ffhir-ehr.sandboxcerner.com%2Fdstu2%2F0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca%2F
 </pre>
 
+#### Redirect URI ####
+
```

Comment:
We should add an error example for this in the troubleshooting section so people know how to fix their issue with multiple redirect URIs situation.

Reply:
Can you elaborate on which error you are referring to ?

Reply:
Sorry for the delay!  I just saw this additional comment.

I was referring to the situation where there are multiple registered redirect URIs but the application isn't sending a redirect URI during the authorize flow.  I assume that's the error that the user is seeing today which led to this update in the documentation.

In this case, what error does the user see?

Reply:
I thought this defaulted to the first listed/primary. @icon5585 - is that what we confirmed?

Reply:
@daliboz that is correct, if no redirect uri is provided then we default to the primary redirect uri that is registered

Reply:
@kolkheang  I think the error that the user saw was because, the redirect URL was not an absolute URL. More of the type : ```abc://``` instead of ```abc://callback```. 

Considering the above I am not adding an error scenario.